### DISCUSSION TITLE
Scale to fit

### DISCUSSION THREAD

**Discussion by ghost** [slidevjs/slidev#790] [Q&A]
This seems like it should be simple.  I feel like I'm missing something.  Is there a simple way to scale content to fit on a slide?  In other words, take all the content between two ''' instances (including images) and fit the content while applying sensible defaults to font sizes, and keeping images from getting clipped at the bottom of the slide.

**Comment by gforien** [comment]
Same question here

1. I tried a naive [embedded style](https://sli.dev/guide/syntax.html#embedded-styles)
```markdown
---
<style>
.slidev-layout { zoom: 150%; }
</style>

Removing DB passwords.
---
```
But the scaling applied to the whole presentation 🤔 
Maybe because I'm trying this on stackblitz.com, not in local

2. I got around by selecting the slide n°2 explicitly (the one I wanted to scale), but there should be a better way to do this
```markdown
---
<style>
.slidev-page-2 { zoom: 150%; }
</style>

Removing DB passwords.
---
```

**Reply by gforien** [reply]
Update !
In local environment, with Slidev v0.40.5 installed globally with npm

I add the naive upscale (1) in the first or second slide
```
<style> .slidev-layout { zoom: 120%;} </style>
```

And the whole presentation is upscaled ✨
